.github/workflows/ci.yaml

```diff
@@ -0,0 +1,41 @@
+name: CI
```

Comment:
Is there a reason for not reusing fastfeedback.yaml?

Reply:
Can if you want. I wasn't sure of the purpose of that file.